### PR TITLE
auto generate pr for olm-bundle

### PR DESCRIPTION
**Pull request by vijtrip2**
Signed-off-by: Vijay Tripathi <vijat@amazon.com>

Issue #, if available: https://github.com/aws-controllers-k8s/community/issues/1045

Description of changes:
* On semver release of ACK service controller, build the olm-bundle artifacts and create PR in `k8s-operatorhub/community-operators` and `redhat-openshift-ecosystem/community-operators-prod` repos
* If `olmconfig.yaml` file is missing from a service controller, a GH issue will be created. [Sample Issue](https://github.com/vijtrip2/ecr-controller/issues/19)
* If olm-bundle generation fails, a GH issue will be created. [Sample Issue](https://github.com/vijtrip2/ecr-controller/issues/20)
* If `ci.yaml` file is missing in upstream repos for an ACK service controller, a GH issue will be created. [Sample Issue](https://github.com/vijtrip2/ecr-controller/issues/22)
* [Sample community-operators PR ](https://github.com/vijtrip2/community-operators/pull/5)
* [Sample community-operators-prod PR](https://github.com/vijtrip2/community-operators-prod/pull/5)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by vijtrip2** [comment]
cc: @acornett21 @trgeiger

File: cd/olm/gh_issue_missing_oh_ci_template.txt [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+4. Mention this issue number in the pull request
+5. When the PR is merged, close this issue
+
+#### Additional Note:
+Make sure that \`operators/ack-$CONTROLLER_NAME/ci.yaml\` file is present in both
+\`k8s-operatorhub/community-operators\` and \`redhat-openshift-ecosystem/community-operators-prod\`
+repositories. If not, follow the same steps as above.
+
+For more guidance, [create an issue](https://github.com/aws-controllers-k8s/community/issues)
+or [reach out on Slack](https://github.com/aws-controllers-k8s/community#help--feedback).
```
Comment (jaypipes) [inline cd/olm/gh_issue_missing_oh_ci_template.txt:17]: Nice. :+1:

File: scripts/lib/gh.sh [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    echo -n "gh.sh][INFO] Updating PR body... "
+    if ! gh pr edit "$__pr_number" -R "$__org_repo" -F "$__pr_body_file_path" >/dev/null ; then
+      echo ""
+      echo "gh.sh][ERROR] Failed to update pull request"
+      return 1
+    fi
+    echo "ok"
+  fi
+  return 0
+}
```
Comment (jaypipes) [inline scripts/lib/gh.sh:113]: Great work on this script for GH stuff. ++

**Review by jaypipes — APPROVED** [review]
Fantastic, Vijay.

**Pull request merged**